### ISSUE TITLE
hard to install

### ISSUE THREAD

**Issue by vahidalvandi** [RyanDaDeng/laravel-google-recaptcha-v3#15]

**Comment by RyanDaDeng** [comment]
@vahidalvandi I’m sorry about it. Can you provide me with some more details? Which laravel version are you using? I’m willing to update it to you easily install the package.

**Comment by RyanDaDeng** [comment]
Is document hard to follow? Or is the whole installation too hard to set up?

**Comment by vahidalvandi** [comment]
have very long document

**Comment by RyanDaDeng** [comment]
> have very long document

Yeah, you are right, I also consider it is too long to read. I will soon re-write and polish it up next week.

Thank you for the suggestion.
